markdown/events/2022/bytesize-inkscape.md

```diff
@@ -25,3 +25,93 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[6:20](https://www.youtube.com/watch?v=0vKhfedYKGo&t=380)
+There are other things you can do with Inkscape. For example these are a couple of other things I've done. This was a a schematic overview of some characteristics of ancient DNA, done all in Inkscape. Actually all of these little Emojis, they have been imported from another project with OpenMoji. Everything is in SVG and it's very easy to drag and drop it into my image, recolor things, change the size and so on, which is really nice. That's all benefits from this vector imaging format! Also you can do more realistic drawings by tracing, which can be done in Inkscape. It's not so cartoony as all the other previous objects. This is a tooth, cut in half, and that's showing how to sample for ancient DNA. You can load the raster image into Inkscape then trace over the top and fill in the colors. As such you can get a particularly good representation of the object, which is then very manipulable after.
+
+[7:23](https://www.youtube.com/watch?v=0vKhfedYKGo&t=443)
+That's the things you can do with Inkscape. Now, what i'd like to do, is to go into Inkscape itself and show you some of the basic functionalities that you would need, in order to make one of the pipeline diagrams, such as this one and this one.
+
+[7:44](https://www.youtube.com/watch?v=0vKhfedYKGo&t=464)
+When you open Inkscape it should look something like this. The UIs (user interface) should be relatively familiar: you've got the toolbar right at the top, you've got other toolbars on the sides as well. It may look a little bit different, depending on what operating system you are working on, but generally this is the outline you will have, at least in the latest version. What you should hopefully see on my screen is a key logger, you should be able to see what I'm typing. In the case I use a shortcut, you can also see that, so keep an eye on that as well.
```

Comment:
```suggestion
When you open Inkscape it should look something like this. The UIs (user interface) should be relatively familiar: you've got the toolbar right at the top, and you've got other toolbars on the sides as well. It may look a little bit different, depending on what operating system you are working on, but generally, this is the outline you will have, at least in the latest version. What you should hopefully see on my screen is a key logger, you should be able to see what I'm typing. In the case I use a shortcut, you can also see that, so keep an eye on that as well.
```